Source/WebKit/UIProcess/API/Cocoa/WKSnapshotConfiguration.mm

```diff
@@ -80,7 +82,16 @@ - (void)_setIncludesSelectionHighlighting:(BOOL)includesSelectionHighlighting
 {
     _includesSelectionHighlighting = includesSelectionHighlighting;
 }
+#endif // PLATFORM(MAC)
 
-#endif
+- (BOOL)_usesTransparentBackground
+{
+    return _usesTransparentBackground;
+}
+
+- (void)_setUsesTransparentBackground:(BOOL)usesTransparentBackground
+{
+    _usesTransparentBackground = usesTransparentBackground;
+}
```

Comment:
This can likely be @synthesize?

Reply:
Sadly, @synthesize is not allowed in a category implementation, and since this is defined in the WKSnapshotConfiguration (WKPrivate) category, we have to fall back to custom getter/setter.